instrumentation/aws-sdk/aws-sdk-1.11/library-instrumentor/build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+plugins {
+  id("otel.library-instrumentation")
+}
+
+base.archivesName.set("${base.archivesName.get()}-instrumentor")
+project.description = "Instruments calls to AWS services made with AWS SDK for Java clients by adding a tracing client as a request handler."
```

Comment:
We don't have descriptions in most of our projects. I'd like to leave them out for this PR rather than having just a couple of libraries with descriptions